packages/pmml-editor/src/editor/components/Outputs/organisms/OutputsContainer.tsx

```diff
@@ -122,6 +142,18 @@ export const OutputsContainer = (props: OutputsContainerProps) => {
     setActiveOperation(Operation.UPDATE_OUTPUT);
   };
 
+  const addBatchOutputs = (outputs: string) => {
+    const outputsNames = outputs.split("\n").filter(item => item.trim().length > 0);
+    dispatch({
+      type: Actions.AddBatchOutputs,
+      payload: {
+        modelIndex: modelIndex,
+        outputFields: outputsNames
+      }
+    });
```

Comment:
dispatching from here for now, I suppose we have to decide a common strategy for this kind of operations.